docs/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -151,12 +151,9 @@ Up to this point, the code shown is just performing regular Http requests, but t
 
 ## Additional resources
 
-- **Using HttpClientFactory in .NET Core**\
-  [*https://docs.microsoft.com/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.1*](/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.1)
-
-- **HttpClientFactory GitHub repo**\
-  [*https://github.com/aspnet/HttpClientFactory*](https://github.com/aspnet/HttpClientFactory)
+- [Using HttpClientFactory in .NET Core](/aspnet/core/fundamentals/http-requests)
+- [HttpClientFactory GitHub repo](https://github.com/aspnet/Extensions/tree/master/src/HttpClientFactory)
```

Comment:
Same here but the link here doesn't need to use italic anymore so it can be simplified

`<https://github.com/aspnet/Extensions/tree/master/src/HttpClientFactory>`